code/element.py

```diff
@@ -0,0 +1,19 @@
+# The element class
+# There should be private variables such as label and names
+
+
+class element:
+
+    label = "" #label is what the user want to be appear on the circuit, it may contains values e.g. 13nC
+    name = "" #name is the element itself
```

Comment:
As in `circuit`, you don't need these lines. You can delete them (and move the documentation to the constructor).